autotst/calculator/statmech.py

```diff
@@ -224,30 +130,11 @@ def write_conformer_file(self, conformer):
         conformer.update_coords_from("ase")
         mol = conformer.rmg_molecule
         output = ['#!/usr/bin/env python',
-                  '# -*- coding: utf-8 -*-', '', 'atoms = {']
-
-        atom_dict = self.get_atoms(conformer=conformer)  # Fix this
-
-        for atom, count in atom_dict.items():
-            output.append("    '{0}': {1},".format(atom, count))
-        output = output + ['}', '']
-
-        bond_dict = self.get_bonds(conformer=conformer) 
-        if bond_dict != {}:
-            output.append('bonds = {')
-            for bond_type, num in bond_dict.items():
-                output.append("    '{0}': {1},".format(bond_type, num))
-            output.append("}")
-        else:
-            output.append('bonds = {}')
-
-        external_symmetry = conformer.calculate_symmetry_number()
+                  '# -*- coding: utf-8 -*-', ]
 
         output += ["",
                    "linear = {}".format(conformer.rmg_molecule.is_linear()),
                    "",
-                   "externalSymmetry = {}".format(external_symmetry),
-                   "",
                    "spinMultiplicity = {}".format(conformer.rmg_molecule.multiplicity),
                    "",
                    "opticalIsomers = 1",
```

Comment:
`linear` and `opticalIsomers` are now optional. Arkane using 3D coords to determine if it's linear (which I think should be more accurate than the `molecule.is_linear()` method which uses 2D graph).  We should probably remove `linear` and `opticalIsomers` as well as the externalSym.